os/overriding-systemd-units.md

```diff
@@ -0,0 +1,105 @@
+# Overriding systemd units
```

Comment:
This title seems focused on one use-case for dropins. Should we reframe it as "Using systemd Drop-In Units" or "Using Drop-In Units to Configure Software" or something like that?